config/crd/bases/redhatcop.redhat.io_groupsyncs.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
@sabre1041 running make install when testing locally reverted this to v0.3.0 not sure what configured this previously. If this is wrong can you advise how to upgrade the makefile?

Reply:
The change you proposed is aok. `v0.4.1` was introduced as I have multiple versions on my machine and the makefile only check to see if `controller-gen` is installed. `v0.3.0` is the version that should be used